Sources/LanguageServerProtocol/SupportTypes/SourceKitFileStatus.swift

```diff
@@ -0,0 +1,31 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2020 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+/// High level categorization of a file status.
+public enum SourceKitFileStatusCategory: Int, Codable, Hashable {
```

Comment:
Unless you have a particular reason to choose Int, I would suggest String and to make this a struct with static fields so that we can extend it with new kinds over time.

Reply:
Done